src/cli/cli_srp_client.cpp

```diff
@@ -258,6 +480,19 @@ template <> otError SrpClient::Process<Cmd("server")>(Arg aArgs[])
     otError           error          = OT_ERROR_NONE;
     const otSockAddr *serverSockAddr = otSrpClientGetServerAddress(GetInstancePtr());
 
+    /**
+     * @cli srp client server
+     * @code
+     * srp client server
+     * [fd00:0:0:0:d88a:618b:384d:e760]:4724
```

Comment:
This line gets dropped from the import, probably because of the [ ]

Can you escape them to see if they come through?

`* \\[fd00:0:0:0:d88a:618b:384d:e760\\]:4724`